src/Symfony/Component/DependencyInjection/Loader/XmlFileLoader.php

```diff
@@ -339,7 +339,11 @@ private function parseDefinition(\DOMElement $service, string $file, Definition
         }
 
         foreach ($this->getChildren($service, 'call') as $call) {
-            $definition->addMethodCall($call->getAttribute('method'), $this->getArgumentsAsPhp($call, 'argument', $file), XmlUtils::phpize($call->getAttribute('returns-clone')));
+            $definition->addMethodCall(
+                $call->getAttribute('method'),
+                $this->getArgumentsAsPhp($call, 'argument', $file),
+                (bool) XmlUtils::phpize($call->getAttribute('returns-clone'))
```

Comment:
I suggest doing a method handling of the case of the missing attribute (instead of handling it as `$call->getAttribute('returns-clone')` returning an empty string which then casts to `false`)

Reply:
Do you want me to basically just do:
```suggestion
                XmlUtils::phpize($call->getAttribute('returns-clone')) !== ''
```
or something else? Because I don't really understand your suggestion.

Reply:
Doing something like `$call->getAttribute('returns-clone') !== '' ? XmlUtils::phpize($call->getAttribute('returns-clone')) : false`, as we need to detect the case of the omitted attribute (to default to `false`) while still using the proper output of `phpize` for cases using the attribute.

Reply:
Done